foundation/foundation-mda/src/main/resources/templates/deployment/hive-metastore-service/hive-metastore-service-values.yaml.vm

```diff
@@ -10,7 +10,7 @@ image:
   imagePullPolicy: IfNotPresent
   dockerRepo: ghcr.io/
   # Overrides the image tag whose default is the chart appVersion.
-  tag: "${versionTag}"
+  tag: "1.7.0"
```

Comment:
I: Locking the version as the v1 chart will be deprecated after 1.7.0

Reply:
We should go ahead and deprecate the profile for the v1 chart in this commit as well.

Reply:
I had intended to deprecate it alongside the v1 SHS and STS charts after completing the full chart (along with writing migration steps, etc).  I'm wary of hitting the deprecation button before the full chart is ready.  I imagine a case where for whatever reason we have to cut a 1.7.1 release and the deprecation is in before the replacement is ready.

Reply:
We'd cut 1.7.1 from 1.7.0 not dev. It's a fair point either way though. I guess I was just thinking if we're refusing to update the image it'd be nice to tell the user why.  But also understand from a user perspective it might be frustrating to see a deprecation message and have no way to resolve it. Could we word it something like "due to ongoing effort to create a Hive v2 chart and subsequent incompatibilities, v1 cannot be upgraded beyond version 1.7" as a middle ground?

Reply:
Absolutely-- I can do that.